models/comment.js

```diff
@@ -44,6 +46,38 @@ CommentSchema.statics.findByAssetId = function(asset_id) {
   return Comment.find({asset_id});
 };
 
+/**
+ * Change the status of a comment.
+ * @param {String} id  identifier of the comment  (uuid)
+ * @param {String} status the new status of the comment
+*/
+CommentSchema.statics.changeStatus = function(id, status) {
+  return Comment.update({'id': id}, {$set: {'status': status}}, {upsert: false}).then(() => {
+    Comment.findById(id).then((comment) => {
+      return comment;
+    }).catch((err) => {
```

Comment:
I usually don't have the `catch` statements here, I put them in the route or wherever the Promise returned from this function is used. Once the error is "caught", you can't do anything else with it. Wherever you implement `Comment.changeStatus` you probably want to send an error back over the network or something in a `catch` statement there.